backend/src/main/scala/cromwell/backend/validation/ValidatedRuntimeAttributesBuilder.scala

```diff
@@ -78,4 +78,5 @@ trait ValidatedRuntimeAttributesBuilder {
     val errorOrListOfKeysToAnys: ErrorOr[List[(String, Any)]] = listOfErrorOrKeysToAnys.sequence[ErrorOr, (String, Any)]
     errorOrListOfKeysToAnys map { listOfKeysToAnys => ValidatedRuntimeAttributes(listOfKeysToAnys.toMap) }
   }
+
```

Comment:
There was a diff above that only took away a blank line and now this only adds one; I'd probably back out both changes.